SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -2,6 +2,7 @@
 HTML renderer
 */
 SCDocHTMLRenderer {
+	classvar <binaryOperatorCharacters = "!@%&*-+=|<>?/";
```

Comment:
it might be good to make this a method of `Symbol::isBinaryOperator`. Then we have a central place to look this up.